docs/scenarios_authoring/AggregatesInTimeWindows.md

```diff
@@ -106,17 +120,18 @@ Parameters specific to the Sliding window:
 
 Parameters specific to the session window:
 
-**endSessionCondition **- the session window  can close not only on timeout; it will also close when the expression entered in this field will evaluate to true. Set it to `false` if the only way to close the window is through session timeout.
+**endSessionCondition** - the session window can close not only on timeout; it will also close when the expression entered in this field will evaluate to true. Set it to `false` if the only way to close the window is through session timeout.
 
 **sessionTimeout**- session window will close after this time since the last event.
 
 **emitWhen** - determines when the event with the result of the aggregation will be emitted. 
 
+Possible values are:
+- `On each event` - Window won't be emitted on end, but after each event. This would be useful e.g. when we want to have daily (i.e. for current day) aggregate for each incoming event, but we're not interested in daily summary on each midnight
```

Comment:
I don't think think description is ok for session window. There's nothing 'daily' or 'midnight' here...